packages/stacked_crashlytics/LICENSE

```diff
@@ -0,0 +1 @@
+TODO: Add your license here.
```

Comment:
Go to the stacked package, copy the licence, make it 2021 for `FilledStacks` and not Dane Mackier